pest_derive/Cargo.toml

```diff
@@ -15,9 +15,9 @@ name = "pest_derive"
 proc-macro = true
 
 [dependencies]
-quote = "^0.3"
-syn = "^0.11"
-pest = { path = "../pest", version = "^1.0" }
+quote = "0.3.15"
```

Comment:
Why not `"^0.3.15"`?

Reply:
Cargo default to using `^`. So both formulations are equivalent. If you would prefer the other form I can change it.

Reply:
Oh, I see. Then, could you please change all of them?

Reply:
I removed all the usages of `^` where they are redundant. It is possible that I misunderstood you, and you wanted me to add them in where they are implied. If so I am sorry, and I can fix it the other way if you ask.

Reply:
There are two other Cargo files that have dependencies in meta and vm.

Reply:
This PR is against the 1.0 branch which does not have those folders.

Reply:
My bad! 😅